pyfr/readers/base.py

```diff
@@ -22,3 +26,209 @@ def to_pyfrm(self):
         mesh['mesh_uuid'] = np.array(str(uuid.uuid4()), dtype='S')
 
         return mesh
+
+
+class BaseMeshConverter(object):
```

Comment:
Okay.  Lets rename this class from BaseMeshReader to something else.  As if we construct an instance of it, rather than inherit, it should probably not be a base class.  Also, it is less of a reader and more of an assembler for taking a bunch of nodes and using these to build up a mesh.  (If we ever support an input file format which already has face connectivity we would not use this class, for example.)